xonsh/environ.py

```diff
@@ -1021,7 +1021,8 @@ def DEFAULT_DOCS():
         ),
         "LANG": VarDocs("Fallback locale setting for systems where it matters"),
         "LS_COLORS": VarDocs(
-            "Color settings for ``ls`` command line utility",
+            "Color settings for ``ls`` command line utility and file arguments "
+            "in subprocess mode with ``$SHELL_TYPE='prompt_toolkit'``",
```

Comment:
It isn't just in prompt-toolkit that this applies

Reply:
The part about file arguments is ptk-specific.  How about:
```suggestion
            "Color settings for ``ls`` command line utility and, "
            "with ``$SHELL_TYPE='prompt_toolkit'``, for file arguments "
            "in subprocess mode",

```

Reply:
That helps!